### PR TITLE
Creates the rectangle button component

### PR DESCRIPTION
**Pull request by JMiller007**
This component solves the issue #4

### CODE REVIEW

File: src/Components/GetStartedButton.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { StyleSheet, Text, View, TouchableOpacity } from 'react-native'
+
+const GetStartedButton = () => {
+    return (
+      <TouchableOpacity
+        style={{
```
Comment (conroyr41) [inline src/Components/GetStartedButton.tsx:7]: This style does not really match our prototype. Here is what I see on my test device:



The button should be a lot wider. The "Inspect" tab on our Figma prototype actually has some style properties that you can copy and paste to make things easier. You can also copy and paste the style properties for the button text.

**Review by conroyr41 — CHANGES_REQUESTED** [review]
Awesome job! The button looks good, I requested just a few changes.

File: src/Components/GetStartedButton.tsx (hunk 1 above)
Comment (christina8711) [reply src/Components/GetStartedButton.tsx:7]: Jordan if you need help making these changes let me know!

Comment (christina8711) [reply src/Components/GetStartedButton.tsx:7]: Requested Changes:
- In the Button Style (After line 7), you can add a height & width property to the Style object.
- In the Text Style (Line 16), change the font size to be the same as what was shown in our Figma prototype.

How I would do it:
- I would first have our Figma prototype pulled up for reference along with our Vs code open: [https://www.figma.com/file/uZyWHhMD4wBsT5ZOnoBmhS/App-Prototype?node-id=2%3A12&t=Q9BekUFKtPoLxvzR-0](url) 
- To view each component of the start button for example, I go to the hierarchy and click on the desired component. Then I look to the tab on the right side of my screen to view the specs such as the sizes of its Width, Height, Font, Border Radius etc.
Screenshot 2023-03-15 at 9 04 25 AM

**Review by conroyr41 — APPROVED** [review]
Awesome work! I just did some very minor changes because the code is indented weirdly and could of been more concise. (Its just nitpicky things lol, so I won't request changes again.) Please see the commits I made above.

**Pull request merged**